### PR TITLE
Node Info

### PR DESCRIPTION
**Pull request by ryanio**
Adds `NodeInfo` to `GethConfig`, along with geth state in redux. This branch relies on https://github.com/ethereum/grid/pull/67

Screen Shot 2019-03-11 at 12 07 06 AM

#### New dependencies? What are they used for?
Adds `classnames, moment, react-minimal-pie-chart` for `NodeInfoBox/Dot` deps

Closes https://github.com/ethereum/grid/issues/35, closes https://github.com/ethereum/grid/issues/73, closes https://github.com/ethereum/grid/issues/74, closes https://github.com/ethereum/grid/issues/38, closes https://github.com/ethereum/grid/issues/75

### CODE REVIEW

**Comment by marcgarreau** [comment]
first render of mui input fields are missing the white background behind the label:
Screen Shot 2019-03-11 at 2 21 43 PM

**Comment by marcgarreau** [comment]
After clicking the toggle "off", the terminal appears to show the node shutting down successfully, but the UI doesnt reflect it in the toggle:
Screen Shot 2019-03-11 at 2 36 16 PM
Clicking the toggle again, the UI shows it turn off and then back on quickly, followed by a :boom::
Screen Shot 2019-03-11 at 2 42 41 PM

**Comment by ryanio** [comment]
ty, should be fixed in latest commit

File: src/API/ReduxStore.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    }
+    default:
+      return state
+  }
+}
+/* eslint-enable */
+
+const store = createStore(mistApp, applyMiddleware(ReduxThunk))
+
+export default store
```
Comment (marcgarreau) [inline src/API/ReduxStore.js]: This file was deleted when the new store (`src/store/index.js`) was created. Is redundant so can be deleted.

File: src/components/Nodes/NodeInfo/NodeInfoDot.jsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    if (NodeInfoDot.isNewBlock(props, this.props)) {
+      let pulseColor
+
+      if (network === 'main') {
+        pulseColor = 'green'
+      } else {
+        pulseColor = 'blue'
+      }
+
+      this.setState({ pulseColor }, () => {
```
Comment (marcgarreau) [inline src/components/Nodes/NodeInfo/NodeInfoDot.jsx]: Getting this error almost immediately after turning on the node this morning:
Screen Shot 2019-03-12 at 9 42 17 AM

File: src/components/NavTabs.js [hunk 3]
```diff
@@ -17,64 +18,73 @@ const styles = theme => ({
...
+            <Tab label="Tools" disabled />
           </Tabs>
         </AppBar>
 
-        {value === 0 && <NodesTab />}
+        <div style={{ display: activeTab === 0 ? 'inherit' : 'none' }}>
+          <NodesTab />
+        </div>
+
+        <TabContainer display={activeTab === 1 ? 'block' : 'none'}>
```
Comment (marcgarreau) [inline src/components/NavTabs.js]: Not a fan of this pattern that affects `display` values and abstracts some of that away into TabContainer. What didn't you like about the conditional rendering? e.g. `{activeTab === 0 && <NodesTab />}`

File: src/store/client/actions.js [hunk 4]
```diff
@@ -1 +1,133 @@
...
+  }
+}
+
+export const startGeth = ({ clientStateManager }) => {
+  return (dispatch, getState) => {
+    const { client } = getState()
+    const { config } = client
+    geth.setConfig(config)
+    geth.start()
+    clientStateManager.start()
```
Comment (marcgarreau) [inline src/store/client/actions.js]: The `geth` API is used throughout the actions file, within the `clientStateManager`, and I think elsewhere too. I believe we should keep these side-effect inducing functions in one place. Will refactor into a service - probably within the clientStateManager, and maybe rename it.

**Pull request merged**